### DISCUSSION TITLE
Barrels

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#187] [General]
I'm fully prepared to be mocked for not knowing this, but was anyone else aware of how dangerous barrels are in the game? I never paid attention to them before, but working on a new level I got in a situation where one of them accidentally got blown up, with jaw-dropping results.

![](https://i.ibb.co/zSkrLwz/Desktop-Screenshot-2024-08-19-23-10-26-78.png)

There are two kinds, purple barrels contain psycho gas, but more interestingly, yellow ones simply explode. And I do mean _explode_, each one goes up with the force of a nuclear grenade the instant it's shot. In the example area above, it causes a massive chain reaction, purging anything nearby.

I never even realised this was possible. It's kind of a shame they're map objects and not level-dependent, as that's pretty much exactly why I never realised how they can be used - they're never in any interesting places in the final game levels.

**Comment by mefistotelis** [comment]
Ahh, Reykjavik. I even remember the game speed drop when exploding these. The gas generates a lot of sprites.
